device.h

```diff
@@ -71,6 +72,10 @@ struct vcam_device {
     char vcam_fb_fname[FB_NAME_MAXLENGTH];
     struct proc_dir_entry *vcam_fb_procf;
     struct mutex vcam_mutex;
+    /* framebuffer info  */
+    struct fb_info info;
+    void *buffer_addr;
```

Comment:
It is a bit confusing between `buffer_addr` and `offset_buffer`. Can you clarify? That is, `fb_addr` and `fb_offset`.